src/tiramisu_core.cpp

```diff
@@ -4163,6 +4163,263 @@ void computation::loop_reversal(int L0)
     this->set_schedule(schedule);
 }
 
+void computation::skewing(tiramisu::var L0_var, tiramisu::var L1_var,
```

Comment:
Please call the function skew and just comment the old skew function (and all of its documentation). This function should replace the old one.